### ISSUE TITLE
Azure run config -> plain text

### ISSUE THREAD

**Issue by joeflack4** [jhu-bids/TermHub#320]
## Overview
Right now the command to start up the app (as well as some possibly necessary system dependency installations) is configured in the Azure portal under configuration > general settings (e.g. [here](https://portal.azure.com/#@live.johnshopkins.edu/resource/subscriptions/fe24df19-d251-4821-9a6f-f037c93d7e47/resourceGroups/jh-termhub-webapp-rg/providers/Microsoft.Web/sites/termhub/slots/dev/configuration) for the dev app).

I believe that Martin said today that it is possible to instead commit a plain text file to TermHub where we store this configuration. This would be the better option.

**Comment by hugwuoke** [comment]
Ohhh did we do this ?

**Comment by joeflack4** [comment]
@hugwuoke Nope, and I don't think we plan on doing it for a long time. It's a low priority. I'm glad we added the `Urgency` field recently, largely on your suggestion. I just marked it appropriately.
